### PR TITLE
Added coverb page to showcase

### PR DESCRIPTION
**Pull request by jhipster-latino**
I added a new case and ordered the rest

### CODE REVIEW

File: pages/showcase.md [hunk 1]
```diff
@@ -18,6 +18,19 @@ If you want your application to be featured here just submit a PR to the [websit
 <br/><br/><br/>
 
 <div class="row">
+    <div class="col-xs-12 col-sm-6 col-md-4">
+        <div class="thumbnail no-margin-bottom">
+            <div class="video-container">
+                <img width="560" height="315" src="http://localhost:4000/images/showcase/coverb-page.png" />
```
[the remark this answers is not in the document]
Comment (jhipster-latino) [reply pages/showcase.md]: Solve! image link point to site.url

**Comment by SudharakaP** [comment]
@jhipster-latino : Hi, can you sign the CLA and resolve the conflicts please :smile:

**Comment by avdev4j** [comment]
@jhipster-latino please fix the conflict and sign the CLA :)

Otherwise, I will close this PR

**Comment by mraible** [comment]
Closing since it's been 17 days.

**Pull request closed without merging**